src/commands/createHttpFunction.ts

```diff
@@ -33,7 +33,6 @@ export async function createHttpFunction(context: IActionContext): Promise<void>
         suppressCreateProjectPrompt: true,
         templateId: 'HttpTrigger',
         languageFilter: /Python|C\#|^(Java|Type)Script$/i,
-        functionSettings: { authLevel: 'anonymous' },
-        targetFramework: ['net6.0', 'net7.0'] // Will only work on functions api v1.4.0, but won't hurt on v1.3.0
+        functionSettings: { authLevel: 'anonymous' }
```

Comment:
Thanks for putting up this PR! Looking a little more into this I am not sure this will be the best solution since there are .NET runtimes that are not supported. We would likely just want to add .NET 8 to the `targetFramework` and then possibly make that the default runtime like suggested in this [comment](https://github.com/microsoft/vscode-azurestaticwebapps/issues/761#:~:text=Now%20with%20.NET%208%2C%20we%20should%20allow%20users%20the%20ability%20to%20choose%20.NET%208%2C%20and%20default%20to%20.NET%208%20(currently%20we%20default%20to%20.NET%206)%20%40nturinski).  Feel free to make those changes yourself or we can try and get to it sometime in the future 😊

Reply:
Ah, that makes sense! Totally understood, I will change this PR to default to .NET 8 as the `targetFramework` for now and continue to look into the HTTP Function selection changes. That will require some change to be done in the VS Code Azure Function Extension to populate the Subwizard for our specific .NET runtime selector then :)

Thank you for your review!